examples/use-cases/cloud_router_aws/main.tf

```diff
@@ -387,36 +339,58 @@ resource "aws_eip" "public_ip_2" {
   tags = {
     Name = "${var.tag_name}-${random_pet.name.id}"
   }
-  depends_on = [
-    aws_instance.ec2_instance_2
-  ]
+}
+
+# Private IPs of the demo Ubuntu instances
+output "ec2_private_ip_1" {
+  description = "Private ip address for EC2 instance for Region 1"
+  value       = aws_instance.ec2_instance_1.private_ip
+}
+
+output "ec2_private_ip_2" {
+  description = "Private ip address for EC2 instance for Region 2"
+  value       = aws_instance.ec2_instance_2.private_ip
+}
+
+# Public IPs of the demo Ubuntu instances
+output "ec2_public_ip_1" {
+  description = "Elastic ip address for EC2 instance for Region 1 (ssh user: ubuntu)"
+  value       = aws_eip.public_ip_1.public_ip
+}
+
+output "ec2_public_ip_2" {
+  description = "Elastic ip address for EC2 instance for Region 2 (ssh user: ubuntu)"
+  value       = aws_eip.public_ip_2.public_ip
 }
 
 # From the PacketFabric side: Create a cloud router
-resource "cloud_router" "cr" {
-  provider     = packetfabric
-  scope        = var.pf_cr_scope
+resource "packetfabric_cloud_router" "cr" {
+  provider = packetfabric
+  scope        = var.pf_cr_scope # Parameter deprecated
   asn          = var.pf_cr_asn
   name         = "${var.tag_name}-${random_pet.name.id}"
   account_uuid = var.pf_account_uuid
   capacity     = var.pf_cr_capacity
   regions      = var.pf_cr_regions
+  lifecycle {
+    ignore_changes = all
```

Comment:
Why should we ingore all changes here?

Reply:
What I found out was if I don't add this, if I run the terraform apply after it has been executed once (end 2 end) it will see modification in the cr and crc resource and will want to destroy and re-create those resources even thought it should not. This actually might be more a bug, I will simplify the use case I wrote and try to repo and open an issue. Will let you know.

Reply:
@medzin so I could not repo it, I think it might have been confused with the output. I am getting correct behavior in my test `Plan: 0 to add, 1 to change, 0 to destroy.` Thanks for pointing this out!

Reply:
I removed it btw.